plugins/__init__.py

```diff
@@ -124,6 +127,20 @@ def get_resource_attribute(
         return Unknown(source=resource)
 
     resource_state = json.loads(resource_state_raw)
+    if resource_state is None:
```

Comment:
I am having second thoughts about dealing with the null state which is never supposed to happen here, isn't it a bit redundant?

Reply:
I'm a bit confused as well. If it should not happen, it should be a hard failure if it does. 

The question is: did it happen? If so, is the assumption that it should not happen valid.

Reply:
It happened before I fixed the behavior in the handler, not after that.  So I will leave it be a hard failure.